grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -781,7 +824,7 @@ class StockMovementService {
                 .inject([:]) {map, item -> map << [(item.id): item]}
 
         Requisition requisition = Requisition.get(data.first()?.requisition_id)
-        def picklistItemsMap = requisition?.getPicklist()?.picklistItems?.groupBy { it.requisitionItem.product.id }
+        def picklistItemsMap = requisition?.getPicklist()?.picklistItems?.findAll { it.pickable }?.groupBy { it.requisitionItem.product.id }
```

Comment:
This feels like a good candidate for a method added to Picklist. Also, getPicklist can be expressed as just "picklist"
```
requisition?.picklist?.pickablePicklistItems
```
```
List<PicklistItem> getPickablePicklistItems() { 
    return picklistItems.findAll { it.pickable }
}
```
```
requisition?.picklist?.pickablePicklistItemsByProduct
```
```
List<PicklistItem> getPickablePicklistItemsByProduct() { 
    return pickablePicklistItemsByProduct?.groupBy { it.requisitionItem.product.id }
}
```